tmt-app/src/Components/MajorPage.jsx

```diff
@@ -1,18 +1,48 @@
-import { Container, Typography, Grid, Card } from "@mui/material";
+import {
+  Container,
+  Typography,
+  Grid,
+  Card,
+  List,
+  ListItemText,
+} from "@mui/material";
 import React from "react";
 import { useParams } from "react-router-dom";
 import { MAJORS_LIST } from "../utils/constants";
 import Breadcrumb from "./Breadcrumb";
 
 function MajorPage() {
   const params = useParams();
-  let data = [];
+  const data = MAJORS_LIST[`"${params.major}"`];
```

Comment:
I found why lol, the constants object for this has doubled the quotes! We should fix that there and we can fix these here